### PR TITLE
Update default sub-team reviewer

### PR DESCRIPTION
**Pull request by liangxia**
1. Add group-lead as approver for sub-teams.
2. Sort approvers/reviewers.

@jhou1 @wsun1 @xltian @akostadinov @pruan-rht  Please help review the PR.

@xingxingxia  @yapei @chengzhang1016 @jhou1 @qinpingli @yasun1 for awareness that you can merge PR by adding /lgtm in specific sub-folders/teams.

### CODE REVIEW

**Comment by jhou1** [comment]
Thank you @liangxia .

File: features/admin/OWNERS [hunk 1]
```diff
@@ -1,6 +1,7 @@
 approvers:
+  - xingxingxia
```
Comment (wsun1) [inline features/admin/OWNERS]: maybe add @chengzhang1016 as another approver?

File: features/logging/OWNERS [hunk 2]
```diff
@@ -1,5 +1,6 @@
 approvers:
+  - chengzhang1016
```
Comment (wsun1) [inline features/logging/OWNERS]: Logging is in Group D, suggest to ask @yapei to be approver

File: features/upgrade/etcd/OWNERS [hunk 3]
```diff
@@ -1,4 +1,4 @@
 approvers:
```
Comment (wsun1) [inline features/upgrade/etcd/OWNERS]: etc is in Group B, suggest to ask @xingxingxia to be approver

**Comment by yasun1** [comment]
@liangxia  No problem for online. For current group-i, the owned service OCM should be another directory `ocm`  but not online(it is only for online-registration). Please help create the directory `ocm`, and the approvers are `xueli181114`, `yasun1`,  the reviewers are `xueli181114`, `yuwang-RH`, `tzhou5`

**Comment by liangxia** [comment]
@wsun1 @yasun1 Added/updated.

**Comment by liangxia** [comment]
The approvers are disabled.

**Comment by liangxia** [comment]
The PR adds indention for reviewers to be more readable, and minor update to several reviewers. @pruan-rht @akostadinov

**Comment by akostadinov** [comment]
this fell between the cracks. Sorry.

**Pull request closed without merging**